hooks/useSocketAssets.ts

```diff
@@ -15,46 +16,63 @@ const ETH = {
   icon: '/img/assets/WETH.svg',
   logoURI: '/img/assets/WETH.svg',
   amount: 0,
+  usdAmount: 0,
 } satisfies AssetBalance;
 
+const VELO = {
+  chainId: 10,
+  address: '0x9560e827aF36c94D2Ac33a39bCE1Fe78631088Db',
```

Comment:
@sebipap get it from the json file

Reply:
what JSON file? we don't use it anymore.

Reply:
`@exactly/protocol/deployments/optimism/VELO.json`